<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationClass</class>
 <widget class="QMainWindow" name="RegistrationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_import">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Import</string>
    </property>
    <widget class="QPushButton" name="pushButton_import">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_accountOfClouds">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>amount:0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_setting">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>ICP Settings</string>
    </property>
    <widget class="QSpinBox" name="spinBox_initialIterations">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_InitialITR">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>initial iteration times</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_NextITR">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>next iteration times</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_nextIteration">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_fixedCloud">
     <property name="geometry">
      <rect>
       <x>148</x>
       <y>25</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_fixedCloud">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>102</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>fixed cloud model</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_movingCloud">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>moving cloud model</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_movingCloud">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>75</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ICPStart">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ICP Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_nextITR">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Next ITR</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_result">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Result</string>
    </property>
    <widget class="QLabel" name="label_R">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotation matrix</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_T">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Translation vector</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_R">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;  | 1 0 0 |&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;R=| 0 1 0 |&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;  | 0 0 1 |&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_T">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;T = | 0 0 0 |&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fitnessScore">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fitness score :</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3D">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>3D Result</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Last iterration time :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>354</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Registration.qrc"/>
 </resources>
 <connections/>
</ui>
